src/app/store/config/features.ts

```diff
@@ -280,6 +280,11 @@ export const features: { [key: string]: FeatureDescriptor } = {
     showInDevMenu: true,
     echoFlagKey: "AREnableFallbackToGeneratedAlertNames",
   },
+  ARUseNewErrorMiddleware: {
+    description: "Use updated error middleware",
+    readyForRelease: false,
+    showInDevMenu: true,
```

Comment:
We need to update this to match Echo's PR artsy/echo#524

```suggestion
    description: "Use updated error middleware",
    readyForRelease: false,
    showInDevMenu: true,
    echoFlagKey: "ARUseNewErrorMiddleware",
```

Reply:
https://artsyproduct.atlassian.net/browse/PHIRE-226

Its gonna be in a different pR 🎉